learnAndCodeFinalProject/src/main/java/org/learnAndCode/Login.java

```diff
@@ -0,0 +1,34 @@
+package org.learnAndCode;
+
+import org.learnAndCode.Database.DBConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Login {
+
+    public static User validateLogin(String username, String password) {
+        String query = "SELECT * FROM dbo.\"User\" WHERE name = ? AND password = ?";
```

Comment:
We should add validation on the input parameters as well, to check if username and password are not null, undefined or empty strings.